content/altinn-studio/reference/data/data-modeling/_index.nb.md

```
@@ -5,40 +5,40 @@ description: Datamodell og Datamodellering i Altinn Studio
 Datamodeller for organisasjoner ligger i Altinn Studio sin [repos][1]-løsning.
 For å få tilgang til disse må man logge inn med Altinn Studio bruker. Om det er første gang du bruker Altinn Studio
 må du [lage en bruker][2].
 
-Datamodellene ligger i `<org>-datamodels`-repo, hvor `<org>` er den organisasjonen du tilhører. 
-F.eks. ligger datamodellene til organisasjonen `Testdepartementet (ttd)` under 
+Datamodellene ligger i `<org>-datamodels`-repo, hvor `<org>` er den organisasjonen du tilhører.
+F.eks. ligger datamodellene til organisasjonen `Testdepartementet (ttd)` under
 https://altinn.studio/repos/ttd/ttd-datamodels.
 
-Lenke til dette repoet finner man i [Altinn Studio Dashboard](/nb/altinn-studio/getting-started/navigation/dashboard) (https://altinn.studio/dashboard). Velg _Rediger_ (blått penn-ikon) for å komme til 
+Lenke til dette repoet finner man i [Altinn Studio Dashboard](/nb/altinn-studio/getting-started/navigation/dashboard) (https://altinn.studio/dashboard). Velg _Rediger_ (blått blyant-ikon) for å komme til
```

markdownlint-cli2 (0.17.2)

Bare URL used
null

(MD034, no-bare-urls)